app/talk/active-users.spec.js

```diff
@@ -4,39 +4,40 @@ import { shallow } from 'enzyme';
 import sinon from 'sinon';
 import ActiveUsers from './active-users';
 
-const activeIds = ['1234', '5678'];
+const activeIds = ['1', '2'];
 const users = [
   { id: '1', display_name: 'Test_User_1', login: 'testUser1' },
   { id: '2', display_name: 'Test_User_2', login: 'testUser2' }
 ];
 
 describe('ActiveUsers', function () {
   let getActiveIdsStub;
-  let fetchUsersSpy;
   let pageCountSpy;
   let boundedPageSpy;
   let userIdSpy;
+  let fetchUsersStub;
   let wrapper;
 
   before(function () {
     getActiveIdsStub = sinon.stub(ActiveUsers.prototype, 'getActiveUserIds').callsFake(() => Promise.resolve(activeIds));
-    fetchUsersSpy = sinon.spy(ActiveUsers.prototype, 'fetchUncachedUsers');
     pageCountSpy = sinon.spy(ActiveUsers.prototype, 'pageCount');
     boundedPageSpy = sinon.spy(ActiveUsers.prototype, 'boundedPage');
     userIdSpy = sinon.spy(ActiveUsers.prototype, 'userIdsOnPage');
+    fetchUsersStub = sinon.stub(ActiveUsers.prototype, 'fetchUncachedUsers').callsFake(() => Promise.resolve(users));
```

Comment:
I'm not super happy that that i'm stubbing this function in the test, ideally this function is refactored and we stub just the API code...but this works and the code is the same as in prod so if folks are ok I don't think this needs to change.